### DISCUSSION TITLE
Have an option to show UR calculated with only hits inside the interquartile range at the results screen and/or during gameplay

### DISCUSSION THREAD

**Discussion by fiji199** [ppy/osu#24615] [Ideas]
This way, random 100s don't massively raise your UR in an otherwise lower UR score. Not saying to get rid of the old one, but an option to have both would be nice.

**Comment by Rekunan** [comment]
I don't think this should be an option, the game already has too many, and I don't think the settings panel would be a good place for this anyway.

I also disagree with the idea, it's misleading for a value called "unstable rate" to not include outliers. Imagine if a play had only 300's in the IQR, and then outside was only 50's.
